client/logic/ui/new_version_checker.gd

```diff
@@ -0,0 +1,111 @@
+extends Node
+
+@export var OWNER: String = "Codename-Recon"
+@export var REPO: String = "Codename-Recon"
+@export var current_version_label: Label
+
+@onready
+var messages: GlobalMessages = $"/root/Messages"
+
+class Version:
+	var major: int
+	var minor: int
+	var patch: int
+	var valid: bool = false
+
+	static var VERSION_REGEX : RegEx = RegEx.create_from_string("v(?<major>[0-9]+)\\.(?<minor>[0-9]+)\\.(?<patch>[0-9]+)")
+
+	func _init(data: String)->void:
```

Comment:
File needs also to be formatted by the formatter. (Missing spaces before and after `->`)